swarmrl/trainers/continuous_trainer.py

```diff
@@ -11,7 +11,7 @@
 
 class ContinuousTrainer(Trainer):
     """
-    Class for the simple MLP RL implementation.
+    Class for the simple MLP RL implementation. Behaves like epsiodic trainer for ResoBee.
```

Comment:
Why not just use the episodic trainer then? This just should never be used with ResoBee in this case, or ResoBee should be adapted to handle continuous training.

Reply:
removed the comment.